sdk/servicebus/test-resources.json

```diff
@@ -68,6 +70,16 @@
       },
       "properties": {}
     },
+    {
+      "apiVersion": "[variables('sbVersion')]",
```

Comment:
If you're set on a second namespace, can you add that to the list of dependencies for the role assignments?